test/hotspot/jtreg/runtime/strictfp/AbstractStrictfpIntMethod60.jcod

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright (c) 2021, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+// Code below is equivalent to:
```

Comment:
If it is equivalent why do we need the jcod file? Presumably there is something here that differs from what the Java source would produce.

Reply:
An "abstract strictfp" method would be rejected at the source level by a Java compiler, hence the jcod files.

Under the proposed language changes in JEP 306, the Java syntax around "strictfp" stays the same, including rejecting "abstract strictfp" method, but the floating-point semantics and defined to be (implicitly) always strict.